sdjournal/read.go

```diff
@@ -38,7 +38,10 @@ type JournalReaderConfig struct {
 	// the array is empty, entries will not be filtered.
 	Matches []Match
 
-	MaxEntryLength uint64 // the max allowable length for an entry (default is 64K)
+	// explicitly set MaxEntryLenght to 0, which means unlimited
+	DisableMaxEntryLength bool
+	// the max allowable length for an entry (default is 64K)
+	MaxEntryLength uint64
```

Comment:
EDIT: sorry, but how do you mimic the systemd function if someone initialize this Config it will default to unlimited, not 64k as the systemd lib